images/image-builder/run.sh

```diff
@@ -5,6 +5,19 @@ export registry_path="${2}"
 
 source /usr/local/bin/dind.sh
 
+# Configuring docker-credential-helper-ecr
```

Comment:
Can you please add this inside the if [[ "${REGISTRY_ENABLED}" == "true" ]]; then block?